src/bindings/python/src/openvino/frontend/tensorflow/graph_iterator.py

```diff
@@ -10,13 +10,22 @@
 
 
 class GraphIteratorTFGraph(GraphIterator):
-    def __init__(self, tf_graph: tf.Graph, share_weights: bool, inner_graph: bool = False):
+    def __init__(self, tf_graph: tf.Graph, share_weights: bool, inner_graph: bool = False,
+                 input_names_map: dict = None, output_names_map: dict = None):
         GraphIterator.__init__(self)
         self.m_graph = tf_graph
         self.m_node_index = 0
         self.m_decoders = []
         self.m_inner_graph = inner_graph
         self.m_share_weights = share_weights
+        if input_names_map is None:
+            self.m_input_names_map = {}
+        else:
+            self.m_input_names_map = input_names_map
+        if output_names_map is None:
+            self.m_output_names_map = {}
+        else:
+            self.m_output_names_map = output_names_map
```

Comment:
```suggestion
        self.m_output_names_map = output_names_map or {}
```

Reply:
Good suggestion, I will do this in this PR if smth serious is found. Otherwise, let me do this in the next PR.

Reply:
Fixed, thanks